themes/default/layouts/community/puluminaries/list.html

```diff
@@ -0,0 +1,58 @@
+{{ define "hero" }}
+    {{ partial "hero" (dict "title" "Puluminaries") }}
+{{ end }}
+
+{{ define "main" }}
+    <section class="max-w-4xl container mx-auto py-8 px-4 text-center text-lg">
+        <p>
+            The Puluminaries program was created to recognize members of the Pulumi community who are experts and leaders in the field of programming and infrastructure, but also
```

Comment:
this should not be past tense. it should state what it "is"

Reply:
Fixed in 1d46ec204a62f8697284257f2c3d78c43e99f982